scripts/e2e-test.sh

```diff
@@ -131,6 +138,23 @@ while (("$#")); do
             exit 1
         fi
         ;;
+    -n | --noimages) # skip building images
+        export EUNOMIA_TEST_SKIP_IMAGES=yes
+        shift
+        ;;
+    -d | --nodeployment) # skip deploying the operator
+        export EUNOMIA_TEST_SKIP_DEPLOYMENT=yes
+        shift
+        ;;
+    -r | --run) # run specific go e2e test
+        if [ -n "$2" ] && [ "${2:0:1}" != "-" ]; then
+            EUNOMIA_TEST_GO_RUN=$2
+            shift 2
+        else
+            echo "Error: Argument for $1 is missing" >&2
+            exit 1
+        fi
+        ;;
```

Comment:
Curious as to why this is done via environment variables vs managing the settings in the code?

Reply:
It's about the scope of the variables. Using `export` allows us to be more flexible within the shell script. The env variables never make it to the parent shell, so there's no concern about it changing anything important.
We also need to use environment variables so that we can control how `make` behaves when called from `.travis.yml`